util/normalization_functions.R

```diff
@@ -344,6 +344,24 @@ SinglePlatformNormalizationWrapper <- function(dt, platform = "array",
   return(norm.list)
 }
 
+all_same <- function(x, my_tolerance = 1e-9){
```

Comment:
I would consider making this a little bit more descriptive, perhaps something like `check_all_same()`?